scripts/missions/acp/01_A_Crystalline_Prophecy.lua

```diff
@@ -35,6 +35,29 @@ mission.sections =
                 end,
             },
 
+            onEventUpdate =
+            {
+                [10094] = function(player, csid, option, npc)
+                    if option == 1 then
+                        -- Parameters 1 and 2 determine if certain NPCs will appear in the CS.
+                        -- Default values of 0 = They appear.
+                        local noVerena = 0
+                        local noSibyl  = 0
+
+                        -- TODO: Fact check this.
+                        if player:hasCompletedMission(xi.mission.log_id.ZILART, xi.mission.id.zilart.WELCOME_TNORG) then
```

Comment:
Verena goes missing during the storyline and then returns, but the point she is returned is unclear to me. Her departure like should coincide with the mission that sends you to stellar fulcrum if i am remembering right.

As for star sybil.. no idea. maybe the rank 10 fight?

Reply:
I'd bet on some other params here effecting the bastok and sandoria parts of the cs

Reply:
I can confirm Aldo mentions Verena JUST going missing during RotZ mission 2 cutscene.
I thought she got killed, though?

Reply:
Coma, but eventually gets better

Reply:
they just never use her in a mission after (writers kinda forgot about her)

Reply:
Also, for the other parameters, I could not find any variation to the cutscene by changing them
I dont recall the characters involved in said portions of the cutscene ever becoming unavailable, though.

Reply:
one of my chars didn't see trion. might have been from toau where he swaps himself with a rabbit